### PR TITLE
Fix/petition states labelling

### PR DESCRIPTION
**Pull request by freesteph**

### CODE REVIEW

File: app/models/petition.rb [hunk 1]
```diff
@@ -72,6 +71,9 @@ class Petition < ActiveRecord::Base
   facet :tagged_in_moderation,         -> { tagged_in_moderation.by_most_recent_moderation_threshold_reached }
   facet :untagged_in_moderation,       -> { untagged_in_moderation.by_most_recent_moderation_threshold_reached }
 
+  facet :collecting_signatures, -> { not_archived.open_state.by_most_popular }
```
Comment (pixeltrix) [inline app/models/petition.rb:74]: This order needs changing - `by_most_recently_published`, i.e. `reorder(open_at: :desc)`

File: app/views/pages/home/_collecting_signatures_petitions.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <ol class="threshold-petitions">
+    <% actioned[:open][:list].each do |petition| %>
+      <li class="petition-item">
+        <h3><%= link_to petition.action, petition_path(petition), class: "threshold-petition-title" %></h3>
+        <p class="intro"><%= t(:"ui.home_page.open_petitions.open_date", date: short_date_format(petition.closed_at)) %></p>
+      </li>
+    <% end -%>
+  </ol>
+  <p><%= link_to petition_count(:open_explanation, actioned[:open][:count]), petitions_path(state: :open), class: "view-all" %></p>
+<% end %>
```
Comment (pixeltrix) [inline app/views/pages/home/_collecting_signatures_petitions.html.erb:14]: Is this just not `_open_petitions.html.erb` ? If so might as well just leave it at that.

File: app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h2><%= link_to petition.action, petition_path(petition) %></h2>
+<p><%= signature_count(:default, petition.signature_count) %></p>
```
Comment (pixeltrix) [inline app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb:2]: Might as well leave out signature counts from the lists

File: app/views/petitions/search/result_items/_petition_result_for_facet_closed.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<h2><%= link_to petition.action, petition_path(petition) %></h2>
+<p><%= signature_count(:default, petition.signature_count) %></p>
```
Comment (pixeltrix) [inline app/views/petitions/search/result_items/_petition_result_for_facet_closed.html.erb:2]: Might as well leave out signature counts from the lists

File: app/views/petitions/search/result_items/_petition_result_for_facet_under_consideration.html.erb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<h2><%= link_to petition.action, petition_path(petition) %></h2>
+<p><%= signature_count(:default, petition.signature_count) %></p>
```
Comment (pixeltrix) [inline app/views/petitions/search/result_items/_petition_result_for_facet_under_consideration.html.erb:2]: Might as well leave out signature counts from the lists

File: config/locales/petitions.en-GB.yml [hunk 6]
```diff
@@ -11,11 +11,9 @@ en-GB:
     facets:
       public:
         - :all
-        - :open
-        - :referred
-        - :debated
-        - :not_debated
-        - :completed
+        - :collecting_signatures
```
Comment (pixeltrix) [inline config/locales/petitions.en-GB.yml:14]: Can you leave in the debated ones - the feature flag switches to the alternate list so I want to be able to bring those back if required on preview

**Review by pixeltrix — CHANGES_REQUESTED** [review]

File: app/views/pages/home/_collecting_signatures_petitions.html.erb (hunk 2 above)
Comment (freesteph) [reply app/views/pages/home/_collecting_signatures_petitions.html.erb:14]: The template is dynamically looked up based on the facet (https://github.com/scottishparliament/e-petitions/blob/fix/petition-states-labelling/app/views/petitions/search/_results.html.erb#L8) 

Do you want me to remove the open one instead?

File: app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb (hunk 3 above)
Comment (freesteph) [reply app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb:2]: Sure, but why?

Comment (freesteph) [reply app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb:2]: Nope, gotcha

Comment (freesteph) [reply app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb:2]: Found it

Comment (freesteph) [reply app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb]: will do all of these in a subsequent PR addressing the specific Trello card asking for it

**Review by pixeltrix — APPROVED** [review]

**Pull request merged**